spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/MeterRegistryCustomizerTests.java

```diff
@@ -55,9 +58,21 @@ public void commonTagsAreAppliedBeforeRegistryIsInjectableElsewhere() {
 				.withUserConfiguration(MeterRegistryCustomizerConfiguration.class)
 				.run((context) -> {
 					MeterRegistry registry = context.getBean(MeterRegistry.class);
-					assertThat(
```

Comment:
The assertion is superfluous in this case. `registry.get` throws an exception if no meter matches.